### PR TITLE
Add flake8 to CI configuration

### PR DESCRIPTION
**Pull request by LucyJimenez**
Include flake8 parameters to the CI file.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -9,5 +9,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Run Lint Tests
-        run: echo "Hello, world!"
\ No newline at end of file
+      - name: flake8
+        run: pip install flake8
+        uses: py-actions/flake8@v2
+        run: flake8 --max-line-length=99 . || true
```
Comment (datapythonista) [inline .github/workflows/ci.yml:15]: Not sure if you already saw it, but this doesn't seem to be working: https://github.com/LucyJimenez/asv/actions/runs/1770088885

Also, can you move the flake8 settings to `setup.cfg` as we discussed please.

Comment (LucyJimenez) [reply .github/workflows/ci.yml]: Yes, I saw it. So, I tried something different and also added the setup.cfg. Please, check the changes.

**Review by datapythonista — CHANGES_REQUESTED** [review]
Looks much better now, almost there, thanks @LucyJimenez 

I guess you've seen the CI error, and yoi know jpw to fix it, just let me know if you need help.

**Review by datapythonista — APPROVED** [review]
Thanks @LucyJimenez, realy nice.

**Pull request merged**